### ISSUE TITLE
Allow password from file

### ISSUE THREAD

**Issue by kapolos** [arangodb/arangodb-docker#31]
Feature request: Add a new variable `ARANGO_ROOT_PASSWORD_FILE` that allows for the password to be passed in as a file. 

Reason: Enables better secrets management by orchestration tools.

Example of others doing it: The official MySQL docker image.

The change should be as simple as adding

```
		if [ -f "$ARANGO_ROOT_PASSWORD_FILE" ]; then
			ARANGO_ROOT_PASSWORD="$(cat $ARANGO_ROOT_PASSWORD_FILE)"
		fi
```

immediately before: https://github.com/arangodb/arangodb-docker/blob/official/docker-entrypoint.sh#L41

**Comment by dothebart** [comment]
Sorry - for some reason I've overlooked this. It will be part of the arangodb 3.1.21 release in the next weeks.

**Comment by dothebart** [comment]
Its merged - so a new 3.1.20 docker image will contain this feature.

**Comment by rm-rf-etc** [comment]
Need to add this to the documentation, and should be mentioned [in the docker hub readme](https://hub.docker.com/r/arangodb/arangodb/)
